src/main/kotlin/racingcarWinner/ui/OutputView.kt

```diff
@@ -1,38 +1,32 @@
 package racingcarWinner.ui
 
-import racingcarWinner.core.Car
-import racingcarWinner.core.MessageCode
-import racingcarWinner.core.Winner
-import racingcarWinner.core.WinnerRacing
+import racingcarWinner.domain.Car
+import racingcarWinner.util.MessageCode
 
 object OutputView {
     private const val MOVE_STEP = "-"
     private const val SEPARATOR = " "
     private const val COMMA = ", "
     private const val COLON = " : "
     private const val ESCAPE_LINE = "\n"
-    private const val ESCAPE_DOUBLE_LINE = "\n\n"
 
-    fun outputResult(cars: List<Car>, winner: Winner, tryNumber: Int) {
-        println(MessageCode.Result.message)
-        print(tryAllRacingOutput(cars, winner, tryNumber))
+    fun outputResult() {
+        println(ESCAPE_LINE.plus(MessageCode.Result.message))
     }
 
-    fun outputWinner(cars: List<Car>, winner: Winner) {
-        val winnerList = winner.getWinner(cars).joinToString(COMMA)
-        println(ESCAPE_LINE.plus(winnerList).plus(MessageCode.WinnerResult.message))
+    fun printTryOneRacingOutput(racingCars: List<Car>) {
+        println(tryOneRacingOutput(racingCars).plus(ESCAPE_LINE))
```

Comment:
```suggestion
     racingCars.forEach {
            println("${it.carName}: ${"-".repeat(it.moveStep)}")
        }
        println()
```

이렇게하면 조금 더 간결하게  표현할 수 있습니다!